src/org/ticket/model/dao/GenericDAO.java

```diff
@@ -0,0 +1,86 @@
+package org.ticket.model.dao;
+
+import java.util.List;
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.persistence.NonUniqueResultException;
+import javax.persistence.Query;
+
+public class GenericDAO<T> implements iGenericDAO<T> {
+
+    EntityManager manager = JPAConn.instance();
+
+    @Override
+    public void save(T object) {
+        manager.getTransaction().begin();
+        manager.persist(object);
+        manager.getTransaction().commit();
+        System.out.println(object.getClass().getSimpleName() + " saved successfully");
```

Comment:
This is a view responsibility so it shouldn't return a void and maybe a boolean instead. That way the callee could take action over the return state, please modify this implementation and all of the others that makes the same thing.

Reply:
I totally agree, in fact, exceptions should now be handled by the caller, not by controllers and DAO classes. I'd like some notes on these changes, since I'm not 100% sure about them.